README.md

```diff
@@ -59,33 +60,89 @@ Table of Contents
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Setup:
-The following `$SHELL` variables should be defined:
+## Setup
 
-- `$BUILD_HOME`
-- `$DATA_PATH`
-- `$ELOG_PATH`
+The `$SHELL` variable `$ELOG_PATH` should be defined:
+
+```
+export ELOG_PATH=/your/favorite/elog/path
+```
+
+Remove and download the setup script:
+
+```
+rm -f setup_gemdaq.sh
+wget https://raw.githubusercontent.com/cms-gem-daq-project/sw_utils/master/scripts/setup_gemdaq.sh
+```
 
 Then execute:
 
 ```
-source $BUILD_HOME/cmsgemos/setup/paths.sh
-source $BUILD_HOME/gem-plotting-tools/setup/paths.sh
+source setup.sh -c 0.3.1 -g 1.0.0 -G 5 -v 2.0.0 -V 3
 ```
 
-If this is the first time you are executing the above commands on `lxplus` it will create a `python v2.7` `virtualenv` for you.  It may take some time to download the necessary packages so be patient and do not interrupt the installation.  To disable this python env execute:
+If this is the first time you are executing the above command, it will create a Python `virtualenv` for you. It will also install the following packages:
+
+* `cmsgemos` version 0.3.1 (`-c 0.3.1`)
+* `gemplotting` version 1.0.0-dev5 (`-g 1.0.0 -G 5`)
+* `vfatqc` version 2.0.0-dev3 (`-v 2.0.0 -V 3`)
+
+It may take some time to download them, so be patient and do not interrupt the installation.
+
+In addition to installing the dependencies, the script will try to guess `$DATA_PATH` based on the machine you are using.
+
+To disable the python env execute:
 
 ```
 deactivate
 ```
 
-To re-enable the python env execute:
+To re-enable the python env, source the script again:
+
+```
+source setup.sh
+```
+
+Note that you should always source the setup script from the same directory.
+
+### Setup at Point 5
+
+Due to the limited Internet access, the setup at Point 5 is slightly more involved.
+
+Create a SOCKS proxy that will allow `pip` to reach the outer world:
+
+```
+PORT=5000
+ssh -D *:$PORT lxplus.cern.ch -N -f
+```
+
+If you get an error saying `bind: Address already in use`, try with `PORT=5001`, `5002`, ...
+
+> **Note**
+> The proxy expires after some time. Just create it again if `pip` complains about the network being unreachable.
+
+Define `$ELOG_PATH`:
 
 ```
-source $BUILD_HOME/venv/slc6/py27/bin/activate
+export ELOG_PATH=/your/favorite/elog/path
 ```
 
-If you are working on a 904 machine, regardless if it is the first time you are executing the above commands or not, the default python `virtualenv` available on the 904 NAS for your operating system (e.g. `cc7` or `slc6`) will be enabled.
+Remove and download the setup script:
```

Comment:
why remove...?  If this is to make sure they have the most up-to-date version add this statement here.  e.g. 

> Remove and download the setup script to ensure you have the most up-to-date version.